docs/core/reconstruction/pattern_recognition/track_finding.md

```diff
@@ -7,6 +7,61 @@ Write CKF documentation
 (ckf_core)=
 ## Combinatorial Kalman Filter
 
+Acts provides a Combinatorial Kalman Filter (CKF) implementation for track finding
+( {class}`Acts::CombinatorialKalmanFilter`).  Its usage is demonstrated in the
+example algorithm ActsExamples::TrackFindingAlgorithm.
+
+The CKF requires as input a track seed i.e. an estimation of the track parameters,
+and a list of measurements with an association to the (sensitive) surfaces of the
+tracking geometry. The CKF propagates track states initialised by the track
+parameter estimates in direction of the momentum (forward propagation).  When ever
+a surface is reached, compatible measurements are search for. In case of multiple
```

Comment:
```suggestion
parameter estimates in direction of the momentum (forward propagation). Whenever
a surface is reached, the CKF searches for compatible measurements. In case of multiple
```